.rhiza/tests/sync/test_rhiza_version.py

```
@@ -0,0 +1,137 @@
+            version_file.unlink()
+
+        # Clear RHIZA_VERSION from environment to test the default value
+        import os
+        import subprocess  # nosec B404
+
+        env = os.environ.copy()
+        env.pop("RHIZA_VERSION", None)
+
+        cmd = ["/usr/bin/make", "-s", "print-RHIZA_VERSION"]
+        logger.info("Running command: %s", " ".join(cmd))
+        proc = subprocess.run(cmd, capture_output=True, text=True, env=env)  # nosec B603
```

Ruff (0.15.14)

[error] `subprocess` call: check for execution of untrusted input

(S603)